app/src/common/shared/com/igalia/wolvic/ui/widgets/settings/SettingsWidget.java

```diff
@@ -454,12 +454,15 @@ public void showView(SettingsView.SettingViewType aType, @Nullable Object extras
                 break;
             case LOGIN_EDIT:
                 if (extras != null) {
-                    showView(new LoginEditOptionsView(getContext(), mWidgetManager, (Login)extras));
+                    showView(new LoginEditOptionsView(getContext(), mWidgetManager, (Login) extras));
```

Comment:
I know it's nice but let's not do this change here, it'll just make git blaming more complicated.

Reply:
Done.